bin/macos/makefile

```diff
@@ -3,17 +3,26 @@
 # SDL2_ttf from https://www.libsdl.org/projects/SDL_ttf/
 # SDL2_net from https://www.libsdl.org/projects/SDL_net/
 
+ARCH := $(shell uname -m)
 SDL_LIB = -L/usr/include -ldl -lm
+MIN_MACOS_VERSION = 10.7
```

Comment:
Move minimal macOS version into a variable as it's used in a couple of places now. Also bumped it from 10.6 to 10.7 as the latest SDL 2.0 release requires 10.7.